extensions/kind/src/create-cluster.ts

```diff
@@ -57,7 +58,19 @@ export async function setupIngressController(clusterName: string) {
   );
 }
 
-export async function connectionAuditor(provider: string): Promise<AuditResult> {
+export async function connectionAuditor(provider: string, image?: string): Promise<AuditResult> {
```

Comment:
I would pass all items as parameter but check only for a given parameter inside the function (so if someone need to verify another parameter we don't change the signature of the method)